Content/0.3.2 - Account Settings/accountSettings.json

```diff
@@ -10,8 +10,6 @@
 
 	"accountSettings_showAssetsSubtitle": "Select which tags to show for assets in this Account",
 	"accountSettings_setBehaviorHeading": "Set how you want this Account to work",
-	"accountSettings_accountSecurity": "Account Security",
```

Comment:
We use this one on Android, should we use **settings_accountSecurityAndSettings** instead ?

Reply:
Yes, the deleted one was in the `accountSettings` space, which is for a completely different screen - Account Preferences. Took me a while to figure out what was going on.

Reply:
It might be that we want it to only show "Account Security" when used as a title. If so, we need to make a new key like `settings_accountSecurityAndSettings_navigationTitle`